engine/src/main/java/com/datatorrent/stram/cli/ApexCli.java

```
@@ -3715,6 +3718,50 @@ DTConfiguration getLaunchAppPackageProperties(AppPackage ap, ConfigPackage cp, L
     return launchProperties;
   }
 
+  private List<AppInfo> getAppsFromPackageAndConfig(AppPackage ap, ConfigPackage cp, boolean onlyConfigApps)
+  {
+    if (cp == null) {
+      return ap.getApplications();
+    }
+
+    File src = new File(cp.tempDirectory(), "app");
+    File dest = new File(ap.tempDirectory(), "app");
+
+    if (!src.exists()) {
+      return ap.getApplications();
+    }
+
+    if (onlyConfigApps) {
+      try {
+        FileUtils.cleanDirectory(dest);
+      } catch (IOException e) {
+        e.printStackTrace();
+      }
+    } else {
+      for (File app: src.listFiles()) {
+        FileUtils.deleteQuietly(new File(dest, app.getName()));
+      }
+    }
+
+    for (File app: src.listFiles()) {
+      try {
+        FileUtils.moveFileToDirectory(app, dest, true);
+      } catch (IOException e) {
+        LOG.error("Application from the config file {} failed while processing", app.getName());
```

Should it be a warn since it's swallowing the exception and moving on?